src/wallet/wallet_args.cpp

```diff
@@ -140,6 +142,23 @@ namespace wallet_args
 
       auto parser = po::command_line_parser(argc, argv).options(desc_params).positional(positional_options);
       po::store(parser.run(), vm);
+
+      if(command_line::has_arg(vm, arg_config_file))
```

Comment:
Huh. Good question... I don't remember to be honest (this is old code). I'll check.